src/app/campaign/creative/creative-card.component.scss

```diff
@@ -17,6 +17,11 @@ h3 {
   justify-content: space-between;
 }
 
+.creative.disabled {
+  color: prx-theme-foreground(disabled-button);
```

Comment:
This color does not have enough contrast on the light background, but it is using the material theme.

Reply:
:+12: i thought the strikethrough made it pretty obvious what was happening.  Silent Files don't get the same treatment ... but those are an edge-case in real life anyways.